main.cpp

```
@@ -1,50 +1,139 @@
 #include "characterClasses/Wizard.h"
-
 namespace fs = std::filesystem;
 
 const char* quest() {
   return "Quest!";
 }
 
-// Main function
-// TODO: a user should be able to pick a class for their character
-int main() {
+bool isValidClass(std::string selectedClass) {
```

Cppcheck (2.10-2)

[performance] Function parameter 'selectedClass' should be passed by const reference.

(passedByValue)